### ISSUE TITLE
Add 20.08 branch

### ISSUE THREAD

**Issue by A6GibKm** [flathub/org.freedesktop.Sdk.Extension.openjdk8#9]
I did some local builds with other versions of openjdk and was unable to build it so it might need non-trivial changes, i.e. more than just switching versions.

**Comment by cxrvh** [comment]
The openjdk11 was already updated & there the issue is explained:
GCC 10 now defaults to -fno-common ([ Provide a 20.08beta version ](https://github.com/flathub/org.freedesktop.Sdk.Extension.openjdk11/commit/e3c2818f62d7c2d1196290bc8cbc6780fd95a163)).

I've created [ branch/20.08 ](https://github.com/cxrvh/org.freedesktop.Sdk.Extension.openjdk8/tree/branch/20.08) with the changes (& some other updates which I also submitted for branch/19.08, #10). But instead of backporting the patches I simply added -fcommon to the flags which seems to be solution some distributions are using (I checked Arch Linux, Debian & Fedora).
